tests/networkpolicy_test.go

```diff
@@ -0,0 +1,208 @@
+package tests_test
+
+import (
+	"flag"
+	"fmt"
+
+	"github.com/google/goexpect"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	v1network "k8s.io/api/networking/v1"
+	v13 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"kubevirt.io/kubevirt/pkg/api/v1"
+	"kubevirt.io/kubevirt/pkg/kubecli"
+	"kubevirt.io/kubevirt/tests"
+)
+
+func assertPingSucceed(ip string, vmi *v1.VirtualMachineInstance) {
+	expecter, err := tests.LoggedInCirrosExpecter(vmi)
+	Expect(err).ToNot(HaveOccurred())
+	defer expecter.Close()
+
+	err = tests.CheckForTextExpecter(vmi, []expect.Batcher{
+		&expect.BSnd{S: fmt.Sprintf("ping -w 3 %s \n", ip)},
+		&expect.BExp{R: "0% packet loss"},
+	}, 60)
+	Expect(err).ToNot(HaveOccurred())
+}
+
+func assertPingFail(ip string, vmi *v1.VirtualMachineInstance) {
+	expecter, err := tests.LoggedInCirrosExpecter(vmi)
+	Expect(err).ToNot(HaveOccurred())
+	defer expecter.Close()
+
+	err = tests.CheckForTextExpecter(vmi, []expect.Batcher{
+		&expect.BSnd{S: fmt.Sprintf("ping -w 3 %s \n", ip)},
+		&expect.BExp{R: "100% packet loss"},
+	}, 60)
+	Expect(err).ToNot(HaveOccurred())
+}
```

Comment:
both functions can combine into one and get a flag to check failed/succeed.

Reply:
I think maybe we can keep it since we call the function separately in below code.